### PR TITLE
Removed ImageFont.getsize and related functions

### PR DESCRIPTION
**Pull request by radarhere**
Suggestion for https://github.com/python-pillow/Pillow/pull/7059

### CODE REVIEW

File: Tests/test_imagefont.py [hunk 1]
```diff
@@ -326,23 +299,17 @@ def test_rotated_transposed_font(font, orientation):
...
-    assert len(log) == 2
     bbox_b = draw.textbbox((20, 20), word)
 
     # Check (w,h) of box a is (h,w) of box b
-    assert box_size_a[0] == box_size_b[1]
-    assert box_size_a[1] == box_size_b[0]
+    assert (bbox_a[2] - bbox_a[0], bbox_a[3] - bbox_a[1]) == (
+        bbox_b[3] - bbox_b[1],
+        bbox_b[2] - bbox_b[0],
+    )
```
Comment (hugovk) [inline Tests/test_imagefont.py:312]: Let's add in a magic comma to make this a bit more readable:
```suggestion
    assert (
        bbox_a[2] - bbox_a[0],
        bbox_a[3] - bbox_a[1],
    ) == (
        bbox_b[3] - bbox_b[1],
        bbox_b[2] - bbox_b[0],
    )
```

File: Tests/test_imagefont.py [hunk 2]
```diff
@@ -373,22 +340,19 @@ def test_unrotated_transposed_font(font, orientation):
...
-    assert len(log) == 1
     bbox_b = draw.textbbox((20, 20), word)
     length_b = draw.textlength(word)
 
     # Check boxes a and b are same size
-    assert box_size_a == box_size_b
+    assert (bbox_a[2] - bbox_a[0], bbox_a[3] - bbox_a[1]) == (
+        bbox_b[2] - bbox_b[0],
+        bbox_b[3] - bbox_b[1],
+    )
```
Comment (hugovk) [inline Tests/test_imagefont.py:355]:
```suggestion
    assert (
        bbox_a[2] - bbox_a[0],
        bbox_a[3] - bbox_a[1],
    ) == (
        bbox_b[2] - bbox_b[0],
        bbox_b[3] - bbox_b[1],
    )
```

**Review by hugovk — APPROVED** [review]
Thanks!

**Comment by radarhere** [comment]
This was removed from https://github.com/python-pillow/Pillow/pull/7059 for simplicity, and has now become part of https://github.com/python-pillow/Pillow/pull/7080

**Pull request merged**